backend/app/services/mongo/explore.py

```diff
@@ -2429,6 +2429,15 @@ async def get_ab_tests_versions(
             }
         },
         {"$unwind": "$task"},
+        {
+            "$lookup": {
+                "from": "event_definitions",
+                "localField": "event_definition.id",
+                "foreignField": "id",
+                "as": "event_def",
+            }
+        },
+        {"$unwind": "$event_def"},
```

Comment:
Please add a comment here : "We look up the last version of the event_definition to have the information up to date"